src/main/ts/rules/NoDirectEditorOptions.ts

```diff
@@ -18,7 +18,7 @@ export const noDirectEditorOptions: Rule.RuleModule = {
     const filename = context.getFilename();
     // Ignore in tests, demos or Options.ts
     // NOTE: To allow for some legacy setups we currently only enforce `Options.ts` instead of `api/Options.ts`
-    if (isPathInTest(filename) || isPathInDemo(filename) || filename.endsWith('/Options.ts')) {
+    if (isPathInTest(filename) || isPathInDemo(filename) || filename.match(/[\/\\]Options\.ts$/)) {
```

Comment:
Same here maybe normalize instead endsWidth reads so much better than a regexp. Or maybe have a PathUtils.pathEndsWith function instead that does normalization and checks.